ROOTFS/etc/waggle/sanity/fatal/static_psu_control.test

```diff
@@ -0,0 +1,159 @@
+acok=$(waggle-get-config -s wagman -k gpio-acok3)
+pg1=$(waggle-get-config -s wagman -k gpio-pg1)
+pg2=$(waggle-get-config -s wagman -k gpio-pg2)
+pg3=$(waggle-get-config -s wagman -k gpio-pg3)
+pg4=$(waggle-get-config -s wagman -k gpio-pg4)
+
+gpiopath="/sys/class/gpio"
+
+
+# not my code, copied from Joe's PSU test
+gpio_high() { 
+
+        gpio=$1
+        if [ ! -f $gpiopath/gpio$gpio/value ]; then 
+                echo "$pf: Cannot find value of device $gpio"
+                exit 1
+        fi
+
+        if [ $(cat $gpiopath/gpio$gpio/value) -eq 1 ]; then 
+                return 0
+        else 
+                return 1
+        fi
+}
+
+# Not my code, taken from Joe's PSU test
+gpio_cleanup()
+{
+        if [ -d $gpiopath/gpio$acok ]; then
+                echo $acok > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg1 ]; then
+                echo $pg1 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg2 ]; then
+                echo $pg2 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg3 ]; then
+                echo $pg3 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg4 ]; then
+                echo $pg4 > $gpiopath/unexport
+        fi
+}
+trap gpio_cleanup EXIT
+
+
+# Not my code, taken from Joe's PSU test
+
+# export the GPIOs and set to input
+if [ ! -d $gpiopath/gpio$acok ]; then
+        echo $acok > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$acok/direction
+if [ ! -d $gpiopath/gpio$pg1 ]; then
+        echo $pg1 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg1/direction
+if [ ! -d $gpiopath/gpio$pg2 ]; then
+        echo $pg2 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg2/direction
+if [ ! -d $gpiopath/gpio$pg3 ]; then
+        echo $pg3 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg3/direction
+if [ ! -d $gpiopath/gpio$pg4 ]; then
+        echo $pg4 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg4/direction
+
+
+# PG2 is the Edge processor, and should correspond to whether it is active in the manifest
+isEPPresent=$(cat /etc/waggle/node_manifest.json | jq .psu.ep_supply.present)
+
+# PG3/PG4 are bound, and should be treated as one AND'ed value. If they aren't AND'ed, big trouble. This AND'ed value should correspond to the peripherals in the manifest.
+isPHPresent=$(cat /etc/waggle/node_manifest.json | jq .psu.ph_supply.present )
+
+# check that we isEPPresent and isPHPresent have values
+
+if [[ $isPHPresent == "" ]];
+then 
+        echo "$pf: Could not get state of PH from manifest FAIL"
+fi
+
+if [[ $isEPPresent == "" ]];
+then 
+        echo "$pf: Could not get state of EP from manifest FAIL"
+fi
+
+
+
+echo "Node EP availability: $isEPPresent"
+echo "Node PH availability: $isPHPresent"
+
+
+bool_to_int() { 
+
+        if $1 -eq "true";
+        then echo "1"
+        else 
+        echo "0"
+        fi
+
+}
+
+
+# PG1 and ACOK Should always be high
+# Error codes 45 and 46 are respectively for failure of ACOK and PG1 
```

Comment:
same comment here about duplicating information in a comment that can be found in the code.  it can get stale and out-of-sync quickly.  From looking at the code people can see that you are using `exit 45` for "ACOK3 not powered".  There is no need to repeat it here in comments.

Comments are intended to give _additional_ information to your code not already provided by the code.